src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -311,6 +311,10 @@ internal void SendInvocationRequest(ScriptInvocationContext context)
                     {
                         if (pair.Value != null)
                         {
+                            if ((pair.Value is DefaultHttpRequest) && IsTriggerMetadataPopulatedByWorker())
```

Comment:
Is there a reason you're using `DefaultHttpRequest` (and using the internal namespace)  instead of the `HttpRequest` abstraction?